backend/contributions/tests/test_steward_permissions.py

```
@@ -421,6 +446,245 @@ def test_steward_cannot_edit_another_stewards_proposal_note(self):
+            ).exists()
+        )
+        notification = Notification.objects.get(recipient=proposer_user)
+        self.assertEqual(notification.event_type, 'submission.proposal_questioned')
+        self.assertEqual(notification.actor, self.steward_user)
+        self.assertIn(str(self.submission.id), notification.link_url)
+
+    def test_review_steward_can_question_more_info_needed_proposal(self):
+        proposer_user = User.objects.create_user(
+            email='proposal-more-info@test.com',
+            address='0x4444444444444444444444444444444444444445',
+            password='testpass123',
```

Ruff (0.15.20)

[error] Possible hardcoded password assigned to argument: "password"

(S106)